std/experimental/allocator/package.d

```diff
@@ -418,13 +586,13 @@ Gets/sets the allocator for the current process. This allocator must be used
 for allocating memory shared across threads. Objects created using this
 allocator can be cast to $(D shared).
 */
-@property IAllocator processAllocator()
+@property shared(ISharedAllocator) processAllocator()
 {
     return _processAllocator;
 }
 
 /// Ditto
-@property void processAllocator(IAllocator a)
+@property void processAllocator(shared ISharedAllocator a)
 {
     assert(a);
```

Comment:
This line is not covered (probably wasn't before, either). You may want to write a simple unittest for it, even with self-assignment.